Makefile

```diff
@@ -0,0 +1,20 @@
+TEMURIN_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk 8u322-b06-jre 11.0.14.1_1-jre
+JOBS := $(addprefix temurin-,${TEMURIN_TAGS})
+CHECKS := $(addprefix check-,${TEMURIN_TAGS})
+CLEAN := $(addprefix clean-,${TEMURIN_TAGS})
+
+all: ${JOBS}
+check: ${CHECKS}
+.PHONY: all ${JOBS}
+
+${JOBS}: temurin-%: Dockerfile
+	docker build \
+	  --build-arg TEMURIN_TAG=$* \
+	  --tag dwolla/docker-java:$*-SNAPSHOT \
+	  .
+
+${CHECKS}: check-%:
+	grep --silent "^          - $*$$" .github/workflows/ci.yml
+
+${CLEAN_JOBS}: clean-%:
```

Comment:
I think `${CLEAN_JOBS}` needs to be `${CLEAN}`, since that's how it's defined on line 4.

Also, It'd be helpful to add a `clean` task so we can run these all at once. (Right now this defines four separate targets, with the tag parameterized into each target name.) We could add

```make
clean: ${CLEAN}
```

and then users could run `make clean` to remove all the snapshot builds. (I'd add it up above, between lines 7 and 8.) That defines a single target `clean` that depends on the other four targets that already exist, so when a user runs `clean`, make will run the other four.